File: seungtae/23_10/programmers/12987.py
import heapq

def solution(A, B):
    answer = 0
    heapq.heapify(A)
    heapq.heapify(B)
    
    while B:
        if heapq.heappop(B) > A[0]:
            heapq.heappop(A)
            answer += 1
    
    return answer

File: seungtae/23_10/programmers/12987.java
import java.util.Arrays;

class Solution {
    public int solution(int[] A, int[] B) {
        int answer = 0;
        Arrays.sort(A);
        Arrays.sort(B);
        
        int aIndex = 0;
        int bIndex = 0;
        
        while (bIndex < B.length) {
            if (B[bIndex] > A[aIndex]) {
                answer += 1;
                aIndex += 1;
            } 
            bIndex += 1;
        }
        
        
        return answer;
    }
}